background/state.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  version: null
```

Comment:
I'd rather separate state-mutating things from the state itself. I'm not sure the `State` we're talking about here is intended the "state" we usually refer to.